examples/saving_and_loading_data/coordinates_in_asdf.py

```diff
@@ -0,0 +1,127 @@
+"""
+========================================
+Saving and Loading Coordinates with asdf
+========================================
+
+In this example we are going to look at saving a loading collections of
+coordinates with `asdf <https://asdf.readthedocs.io/en/latest/>`__.
+
+asdf if a modern file format designed to meet the needs of the astronomy
+community. It has deep integration with Python and SunPy and Astropy as well as
+implementations in other languages. It can be used to store known Python
+objects in a portable, well defined file format. It is primarily useful for
+storing complex Astropy and SunPy objects in a way that can be loaded back into
+the same form as they were saved.
+
+.. note::
+    This example requires Astropy 3.1 and asdf
```

Comment:
Might want to explicitly say `asdf 2.3.0` since that's the minimum version required by Astropy 3.1.